src/core/components/Navbar.js

```diff
@@ -24,62 +24,62 @@ import ShareIcon from '@material-ui/icons/Share';
 import AddIcon from '@material-ui/icons/Add';
 import TextField from '@material-ui/core/TextField';
 import Skeleton from '@material-ui/lab/Skeleton';
-import {Link} from "react-router-dom";
+import { Link } from "react-router-dom";
 
 class Navbar extends React.Component {
-    render () {
+    render() {
         let companies = [];
         if (this.props.companies) {
-            companies.push({name: "All", img: 'logo', id: SEARCH.COMPANY_ALL});
-            companies.push({name: "Personal", img: 'user', id: SEARCH.COMPANY_MY});
-            companies.push({name: "Shared", img: 'share', id: SEARCH.COMPANY_SHARED});
+            companies.push({ name: "All", img: 'logo', id: SEARCH.COMPANY_ALL });
+            companies.push({ name: "Personal", img: 'user', id: SEARCH.COMPANY_MY });
+            companies.push({ name: "Shared", img: 'share', id: SEARCH.COMPANY_SHARED });
             companies = companies.concat(this.props.companies);
 
             companies = companies.map(company => (
                 <ListItem
-                    style={company.id === this.props.activeCompanyId ? {backgroundColor: '#b3b3b3'} : {}}
+                    style={company.id === this.props.activeCompanyId ? { backgroundColor: '#b3b3b3' } : {}}
                     component={Link}
                     to={"/?company=" + company.id}
-                    onClick={() => this.onChangeCompany(company.id)}
+                    onClick={() => this.onChangeCompany(company.id) || this.props.setAddButtonStatus(0)}
```

Comment:
I am not sure, that it's a good pattern:) I think better to don’t use OR in this case